README.md

```diff
@@ -31,6 +32,8 @@ To make use of these trino adaptations in your dbt project, you must do two thin
     dispatch:
       - macro_namespace: dbt_utils
         search_order: ['trino_utils', 'dbt_utils']
+      - macro_namespace: dbt_date
+        search_order: ['dbt_trino_date', 'dbt_date']
```

Comment:
nit; trino_date seems more consistent with the existing naming convention

Reply:
Actually it was a documentation bug, as it needs to be `trino_utils`, as this is the name of dbt project of entire `dbt-trino-utils` packgage, so this is a common namespace for all `dbt-trino-utils` shims.